cmd/copyEnumeratorInit.go

```diff
@@ -112,9 +111,30 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 	// Giving it nothing to work with as new names will be added as we traverse.
 	var containerResolver = NewS3BucketNameToAzureResourcesResolver(nil)
 	existingContainers := make(map[string]bool)
-	var logDstContainerCreateFailureOnce sync.Once
 	seenFailedContainers := make(map[string]bool) // Create map of already failed container conversions so we don't log a million items just for one container.
 
+	createContainerFailureTransfer := func(dstContainerName, failureReason string) error {
+		// Don't spam logs on failed containers
+		if _, ok := seenFailedContainers[dstContainerName]; !ok {
+			// Schedule a intentionally failing transfer at the container level with a empty dummy object
+			dummyObj := newStoredObject(nil, "", "", time.Now(), 0, nil, azblob.BlobNone, dstContainerName)
```

Comment:
I think you'll need to tweak all your usage of newStorageObject because commit 5241069e8918 has changed it signature.  (As noted in aother comment, I'd also suggest centrolizing them in one place).  Pls ping me for re-review once that's done, since I've only done a quick skim-read today.